.github/workflows/cpp.yml

```diff
@@ -128,6 +128,8 @@ jobs:
       ARROW_GANDIVA: ON
       ARROW_HDFS: ON
       ARROW_HOME: /usr/local
+      # TODO(ARROW-16102): Enable this once we can figure out builds.
+      ARROW_GCS: OFF
```

Comment:
yes, [ARROW-16102](https://issues.apache.org/jira/browse/ARROW-16102) was fixed in between this was posted for review and it got reviewed.  this will be removed in the rebase.